src/Temporalio/Worker/WorkflowInstance.cs

```diff
@@ -1459,21 +1450,22 @@ private string GetStackTrace()
             }
             workflowComplete = completionCommandIndex >= 0;
 
-            // This only applies for successful activations that have a completion not at the end,
-            // don't have completion reordering disabled, and that are either not replaying or have
-            // the flag set.
+            // In a previous version of .NET, if this was a successful activation completion with
```

Comment:
nit: (there might be a few other places in the repo)

```suggestion
            // In a previous version of .NET SDK, if this was a successful activation completion with
```

Reply:
A bit pedantic for a code comment, but will change

Reply:
If a reviewer prefixes their comment with "nit", you can't complain it's pedantic :) Universal law of code review!

Reply:
Very true, I definitely missed that prefacing. My bad (and fixed anyways).